### PR TITLE
Blueprints: Add page for customers to submit their own flows

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Add a CTA on the Blueprint page to "Submit Your Own" (CTA to be checked over with @gstout52) 

Screenshot 2024-10-18 at 17 04 56

Also adds a placeholder page for "Submit Your Blueprint". 

- Need to configure the correct form in Hubspot
- Check over the language/content with @gstout52, in particular, what we want this text to say:
Screenshot 2024-10-18 at 17 03 52

## Related Issue(s)

Merge after #2683 
Part of https://github.com/FlowFuse/website/issues/2638

### CODE REVIEW

**Comment by joepavitt** [comment]
Screenshot 2024-10-22 at 16 16 26

@Yndira-E I'm thinking we may want a graphic of some kind on that left-side to jazz it up a little? 

The form is long, and we need the informative text, so I went split view.

**Comment by joepavitt** [comment]
> Were you aiming for it to resemble how the blueprints page will eventually look?

I was, and I agree with your assessment

**Comment by Yndira-E** [comment]
> Don't think it needs to block anything here though as a first iteration

I agree

**Comment by joepavitt** [comment]
@gstout52 you can check out deployment preview here: https://deploy-preview-2684--flowforge-website.netlify.app/

File: src/blueprints/submit.njk [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  <div class="w-full py-10 text-center ">
+    <div class="container m-auto max-w-lg md:max-w-4xl">
+        <h2>Existing Blueprints</h2>
+        <p>Here are a few examples of Blueprints from our collection.You can take a look at the full collection in our <a href="/blueprints">Blueprint Library</a></p>
+        <ul class="flex flex-wrap">
+            {%- asyncEach item in blueprints -%}
+            <li class="w-full md:w-1/3 my-2 px-3 pb-6 border-b mt-6 flex flex-col justify-between group">
+                <a href="{{ item.url }}" class="relative w-full flex flex-col hover:no-underline">
+                    <div class="ff-blog-tile overflow-hidden">
+                        <div class="w-full h-auto transition-transform group-hover:scale-105">
```
Comment (Yndira-E) [inline src/blueprints/submit.njk:56]: On hover, the corners become sharp again, I've created some classes that I've been using to address this
```suggestion
                        <div class="w-full h-auto transition-transform ff-image-cover ff-image-rounded scale">
```

Screenshot 2024-10-23 at 12 05 55

File: src/blueprints/submit.njk [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                            {% tileImage item, "./images/og-blog.jpg", "Image with logo and the slogan: Elevate Node-RED with Flowfuse", 285 %}
+                        </div>
+                        <label class="absolute w-full left-0 bottom-0 bg-indigo-700 text-white font-medium text-left p-2">{{ item.data.title }}</label>
```
Comment (Yndira-E) [inline src/blueprints/submit.njk:59]: And this one adds round corners to the title background
```suggestion
                        <label class="absolute w-full left-0 bottom-0 bg-indigo-700 text-white font-medium text-left p-2 rounded-b-lg">{{ item.data.title }}</label>
```

**Comment by Yndira-E** [comment]
And this:
Screenshot 2024-10-23 at 11 55 06

It should look like this:
Screenshot 2024-10-23 at 11 55 19

I know this class is causing the issue:
Screenshot 2024-10-23 at 11 55 27

But it seems like you didn't introduce it in this PR, so I'm not sure which of your changes is causing this to happen.

**Review by gstout52 — APPROVED** [review]
Looks great! Let's make the Job Title field mandatory. This will be useful for users browsing the page and our own outreach efforts.

**Comment by sumitshinde-84** [comment]
@joepavitt, I think we need to add one more field to upload the `package.json` file that will include the nodes the blueprint is using.

**Comment by joepavitt** [comment]
@sumitshinde-84 we can form one ourselves here, they're going to want to export from Node-RED, and having to generate a `package.json` is friction we can get them to avoid. We have their e-mail address to be able to check which nodes are used, etc. if they do not detail it in the existing form submission.

**Pull request merged**